### ISSUE TITLE
video/audio elements with controls set should be focusable by default

### ISSUE THREAD

**Issue by chaals** [w3c/html#483]
This change is included in #482 - it is a set of recommendations for defaults, "modulo platform conventions", so does not actually determine conformance.

The following tests for [`audio`](http://butlerccwebdev.net/support/html5-video/audio-demo.html) and [`video`](http://shapeshed.com/examples/HTML5-video-element/) elements show that in multiple browsers this is already the case.

**Comment by stevefaulkner** [comment]
> This change is included in #482 - it is a set of recommendations for defaults, "modulo platform conventions", so does not actually determine conformance.

fixed by #482
